### DISCUSSION TITLE
Design System

### DISCUSSION THREAD

**Discussion by Oluwasetemi** [keystonejs/keystone#6553] [Event Q&A Q3 2021]
Does the team have any plans to fully document the `keystone-ui` design system/guide. I mean the keystone-ui components.

**Comment by raveling** [comment]
@Oluwasetemi great suggestion! 
We answered this question in the recent community Q&A event:
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=1396s

**Reply by Oluwasetemi** [reply]
Thanks. I miss the meeting. Thank you for your response.

**Reply by raveling** [reply]
You're welcome 🙇
